torch_model_base.py

```diff
@@ -648,3 +651,33 @@ def __repr__(self):
         param_str = ["{}={}".format(a, getattr(self, a)) for a in self.params]
         param_str = ",\n\t".join(param_str)
         return "{}(\n\t{})".format(self.__class__.__name__, param_str)
+
+    def _get_set(self,get, set = None):
```

Comment:
Why does this function and `retrieve_activations` needed to be added to the base class? I would prefer to add it to the IIT-specialized classifier or a separate base class for IIT models.